test/webpack-integration.test.js

```diff
@@ -42,87 +43,43 @@ describe('Webpack Integration Tests', () => {
   });
 
   it('calls cssProcessor with correct arguments', (done) => {
-    fs.readFile(__dirname + '/default/default.css', (err, data) => {
-      const expectedCss = data.toString();
-      const destination = 'tmp.css';
-      const cssProcessor = {
-        process: (css, options) => {
-          const assembledOptions = Object.assign({ from: destination, to: destination }, options);
-          expect(options).toEqual(assembledOptions);
-          expect(css).toEqual(expectedCss);
-          return Promise.resolve({ css });
-        }
-      };
-      const cssProcessorOptions = { discardComments: { removeAll: true } };
-      const plugin = new OptimizeCssAssetsPlugin({
-        cssProcessor,
-        cssProcessorOptions
-      });
-      const config = Object.assign(defaultConfig, {plugins: [plugin, new ExtractTextPlugin(destination)]});
-      webpack(config, (err, stats) => {
-        if (err) return done(err);
-        if (stats.hasErrors()) return done(new Error(stats.toString()));
-        done();
-      });
+    const destination = 'tmp.css';
+    const expectedCss = fs.readFileSync(__dirname + '/util/default.css').toString();
+    const cssProcessor = {
+      process: (css, options) => {
+        const assembledOptions = Object.assign({ from: destination, to: destination }, options);
```

Comment:
actually looking back, it looks like you can specify `cssProcessorOptions` which should get merged into the options. maybe we assert that?

Reply:
the thing is the destination stuff gets added to that options object by the plugin as part of the processing. since we don't really care about that, it seems like `expect.objectContaining` will do the trick